docs/notebooks/plot_slopes.ipynb

```diff
@@ -0,0 +1,2503 @@
+{
```

Comment:
<p>great to have this comparison!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/701/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Agreed. It really highlights the value-add of <code>plot_slopes</code></p>